src/Ninja/G1CP/Content/Misc/item.d

```diff
@@ -1,3 +1,12 @@
+/*
+ * Change the value of an item's int variable. Returns true on success.
+ */
+func int G1CP_ItemChangeIntVar(var string name, var string prop, var int originalInt, var int correctedInt) {
+    var int itemId; itemId = G1CP_GetItemInstId(name);
+
+    return (G1CP_ReplaceAssignInt(itemId, 0, ConcatStrings("C_Item.", prop), 0, originalInt, correctedInt) > 0);
```

Comment:
I think this is really good. One thing we should consider is the the item also has integer arrays (`C_ITEM.count[5]`). In order to be able to replace, say `count[3]` specifically, we would either have to expose one more argument (as `G1CP_ReplaceAssignInt` does it) - maybe much better, parse the `prop` name to infer it:

```D
G1CP_ItemChangeIntVar("My_Item_Instance", "count[5]", oldVal, newVal);
```

For that we'd need to find an efficient way to identify and extract the number from the `prop` otherwise that might be too expensive and slow down applying the fixes.